diff --git a/src/data/interlocutors.js b/src/data/interlocutors.js
index 3fd4e9a01..5f674037a 100644
--- a/src/data/interlocutors.js
+++ b/src/data/interlocutors.js
@@ -920,6 +920,11 @@ export const interlocutorAvatars = [
     name: "Blake Giunta",
     src: `${avatarBasePath}/blake-giunta.webp`,
     aliases: ["Blake Giunta"]
+  },
+  {
+    name: "Douglas Wilson",
+    src: `${avatarBasePath}/douglas-wilson.webp`,
+    aliases: ["Douglas Wilson", "Doug Wilson"]
   }
 ];
